tensorflow/python/ops/math_ops.py

```diff
@@ -3212,6 +3212,13 @@ def sigmoid(x, name=None):
 
   Returns:
     A Tensor with the same type as `x`.
+  
+  Usage Example:
+  
+  >>> x = tf.constant([-128.0, 0.0, 128.0], dtype=tf.float32)
+  >>> x = tf.sigmoid(x)
```

Comment:
Hello, Good Work! Just need 1 change for the Ubuntu CPU check to pass. Your code did not printed out the output of tf.sigmoid(x) you can fix it just by removing `x =` cheers :)

JuanC

Reply:
Got it, thanks!